stores/GlobalPath/GlobalPathReducers.ts

```diff
@@ -0,0 +1,27 @@
+import BaseReducer from "@/utils/BaseReducer";
+import GlobalPathActions from "./GlobalPathActions";
+import { GlobalPathState } from "./GlobalPathTypes";
+import { AnyAction } from "redux";
+
+export default class GlobalPathReducer extends BaseReducer {
+    initialState: GlobalPathState = {
+        data: {
+            GlobalPath: []
```

Comment:
Ah the variable here should still be `waypoints` as `data` corresponds to one GlobalPath document = `{waypoints: []`}